### PR TITLE
Docs for usage-based billing

### LINKED ISSUE #2684: Document usage-based billing for teams
The goal of this page is to explain the basics of usage-based billing, so that it makes sense for early access teams.

The page does not need to cover all edge cases like choosing your billing account team when you are a member of multiple teams.

### PR DESCRIPTION
**Pull request by jldec**
## Description
Adds early-access usage-based billing page with images.

- [x] "Usage-based Billing" menu item 
- [x] "Usage-based Billing" page 
- [x] "Beta" label (pending #2682)
- [x] Introduction to usage-based billing and and explanation of Credits
- [x] Configure team billing
- [x] Configure a usage limit
- [x] Good to know

## Related Issue(s)
Fixes #2684 

## How to test
https://deploy-preview-2685--gitpod-kumquat.netlify.app/docs/usage-based-billing

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2685"></a>

### CODE REVIEW

File: src/lib/contents/docs/menu.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export const MENU: MenuEntry[] = [
     M("Collaboration & Sharing", "sharing-and-collaboration"),
     M("Teams & Projects", "teams-and-projects", [], "beta"),
     M("Team Billing", "team-billing", [], "beta"),
+    M("Usage-based Billing", "usage-based-billing", [], "beta"),
```
Comment (Siddhant-K-code) [inline src/lib/contents/docs/menu.ts:43]: In the [screenshot](https://github.com/gitpod-io/website/blob/23455e6a16caeb4fd96729d3afbdadd20156bb15/static/images/docs/billing/active-team-billing.png), we write it as `Usage-Based Billing` (basically in webApp. But, here it is written as `Usage-based Billing` 🤔

Comment (Siddhant-K-code) [inline src/lib/contents/docs/menu.ts:43]: In the [screenshot](https://github.com/gitpod-io/website/blob/23455e6a16caeb4fd96729d3afbdadd20156bb15/static/images/docs/billing/active-team-billing.png), we write it as `Usage-Based Billing` (basically in webApp). But, here it is written as `Usage-based Billing` 🤔 


```suggestion
    M("Usage-Based Billing", "usage-based-billing", [], "beta"),
```

File: gitpod/docs/usage-based-billing.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## View usage details
+
+Team owners can view usage details for their team for the current (unbilled) billing period, and for previous billing periods.
+
+![Active Team Billing](../../../static/images/docs/billing/view-team-usage-details.png)
+
+## Good to know
+
+- Usage-based billing is an early-access (invitation only) feature.
```
Comment (Siddhant-K-code) [inline gitpod/docs/usage-based-billing.md:56]: Same as [this comment](https://github.com/gitpod-io/website/pull/2685/files#r967247895)

```suggestion
- Usage-Based billing is an early-access (invitation only) feature.
```

File: src/lib/contents/docs/menu.ts (hunk 1 above)
Comment (jldec) [reply src/lib/contents/docs/menu.ts:43]: Thanks @Siddhant-K-code 

I'm going by this [guideline](https://titlecaseconverter.com/blog/capitalizing-hyphenated-words/) treating the word "based" in "Usage-based" as a minor word. The text in the screenshot should be fixed to conform to that as well :) 

> The first word in a hyphenated compound is capitalized, and the subsequent words are lowercased if they are minor words (in the respective style), and capitalized else

File: gitpod/docs/usage-based-billing.md (hunk 2 above)
Comment (jldec) [reply gitpod/docs/usage-based-billing.md:56]: see [comment above](#discussion_r967260287)

File: gitpod/docs/usage-based-billing.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+section: develop
+title: Usage-Based Billing
```
Comment (Siddhant-K-code) [inline gitpod/docs/usage-based-billing.md:3]: In respect of [this comment](https://github.com/gitpod-io/website/pull/2685#discussion_r967260287), i think it should also be updated to `Usage-based Billing`

Comment (jldec) [reply gitpod/docs/usage-based-billing.md:3]: Followup PR #2713

**Pull request merged**